README.md

```diff
@@ -87,18 +64,18 @@ risks introduced are outweighed by the functionalities they support.
 
 ![image](/images/install05.png)
 
-### Authentication with PAT (optional)
-Create a Personal Access Token (PAT) for authentication and save the token value as a repository secret. Review [Personal Access Token (PAT) Requirements and Risks](#personal-access-token-pat-requirements-and-risks) before using a PAT.
-
-1. [Create a Personal Access Token](https://github.com/settings/tokens/new?scopes=public_repo,read:org,read:repo_hook,read:discussion) with the following read permissions:
-    - Note: `Token for OSSF Scorecard Action - myorg/myrepo` (Note: replace `myorg/myrepo` with the names of your organization and repository so you can keep track of your tokens.)
-    - Expiration: `No expiration`
-    - Scopes:
-        * `repo > public_repo`                  Required to read [Branch-Protection](https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection) settings. **Note**: for private repositories, you need scope `repo`.
-        * `admin:org > read:org`                Optional: not used in current implementation.
-        * `admin:repo_hook > read:repo_hook`    Optional: needed for the experimental [Webhook](https://github.com/ossf/scorecard/blob/main/docs/checks.md#webhooks) check.
-        * `write:discussion > read:discussion`  Optional: not used in current implementation.
-
+### Authentication with Fine-grained Personal Access Token (optional)
+Create a fine-grained Personal Access Token (PAT) for authentication and save the token value as a repository secret.
+
+1. [Create a fine-grained Personal Access Token](https://github.com/settings/personal-access-tokens/new) with the following settings:
+    - Token name: `OpenSSF Scorecard Action - $USER_NAME/$REPO_NAME>` (Note: replace `$USER_NAME/$REPO_NAME` with the names of your organization and repository so you can keep track of your tokens.)
+    - Expiration: Set `Custom` and then set the date to exactly a year in the future (the maximum allowed)
+    - Repository Access: `Only select repositories` and select the desired repository. Alternatively, set `All repositories` if you wish to use the same token for all your repositories.
+    - Repository Permissions:
+        * `Administration: Read-only`: Required to read [Branch-Protection](https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection) settings.
```

Comment:
Good point! I just checked [the docs](https://docs.github.com/en/rest/overview/permissions-required-for-fine-grained-personal-access-tokens?apiVersion=2022-11-28#administration) and a read token can get a bunch of info on the repo's settings. But only one that's concerning (possibly very concerning!): an Administration-read token can [fetch all deploy keys stored in the repo](https://docs.github.com/en/rest/deploy-keys?apiVersion=2022-11-28#list-deploy-keys).

I'll add this to the documentation and specifically call out the risk to deploy keys and that projects that have such keys should consider whether the risk is worth it.

On a positive note, "Administation read" is also the only thing necessary to fetch information on the repo's default workflow permissions, which would also solve https://github.com/ossf/scorecard/issues/2556 (when run via the Action).

Reply:
This is the public key portion of the deploy key right? 
https://docs.github.com/en/authentication/connecting-to-github-with-ssh/managing-deploy-keys#deploy-keys
> an SSH key that grants access to a single repository. GitHub attaches the public part of the key directly to your repository instead of a personal account, and the private part of the key remains on your server

So is this just similar to what GitHub does with anyone being able to look up a user's public ssh keys?
https://api.github.com/users/spencerschrock/keys

Reply:
I honestly don't know? I would hope so, but I've never actually seen such keys in action for deployments, so I don't even know how to check.

Reply:
I've added the disclaimer mentioning the keys, but it'd be great to discover whether these are just the public keys, because then that is indeed harmless and can be removed.

Reply:
Ok, yeah, I just re-read the start of the [deploy key docs](https://docs.github.com/en/rest/deploy-keys?apiVersion=2022-11-28) and it now seems clear to me that this is just the public key.